app/components/search/Header/Header.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from 'react';
```

Comment:
This component was extracted from the searchResultsPage and serviceDiscoveryResults page below. The only changes involve the QR code modal